### PR TITLE
Adding asce_version parameter

### PR DESCRIPTION
**Pull request by micheles**
We need to change the AELO web interface by setting the default by using
```python
from openquake.commonlib.oqvalidation import OqParam
OqParam.asce_version.default  # 'ASCE7-16'
```

### CODE REVIEW

File: openquake/commands/mosaic.py [hunk 1]
```diff
@@ -168,7 +168,8 @@ def from_file(fname, mosaic_dir, concurrent_jobs):
 
 def run_site(lonlat_or_fname, mosaic_dir=None,
              *, hc: int = None, slowest: int = None,
-             concurrent_jobs: int = None, vs30: float = 760):
+             concurrent_jobs: int = None, vs30: float = 760,
+             asce_version: str = 'ASCE7-16'):
```
Comment (micheles) [inline openquake/commands/mosaic.py:172]: Please change to
```python
asce_version: str = oqvalidation.OqParam.asce_version.default
```

File: openquake/commands/mosaic.py [hunk 2]
```diff
@@ -198,6 +200,8 @@ def run_site(lonlat_or_fname, mosaic_dir=None,
 run_site.slowest = 'profile and show the slowest operations'
 run_site.concurrent_jobs = 'maximum number of concurrent jobs'
 run_site.vs30 = 'vs30 value for the calculation'
+run_site.asce_version = (
+    f'one of {oqvalidation.OqParam.asce_version.validator.choices}')
```
Comment (micheles) [inline openquake/commands/mosaic.py:204]: Please change to
```
python
run_site.asce_version = dict(help=f'one of {oqvalidation.OqParam.asce_version.validator.choices}',
                                                  choices=oqvalidation.OqParam.asce_version.validator.choices)
```

File: openquake/commands/mosaic.py (hunk 1 above)
Comment (ptormene) [reply openquake/commands/mosaic.py]: Done

File: openquake/commands/mosaic.py (hunk 2 above)
Comment (ptormene) [reply openquake/commands/mosaic.py]: Done

File: openquake/server/views.py [hunk 3]
```diff
@@ -864,6 +866,15 @@ def aelo_validate(request):
     except Exception as exc:
         validation_errs[AELO_FORM_PLACEHOLDERS['siteid']] = str(exc)
         invalid_inputs.append('siteid')
+    try:
+        asce_version = request.POST.get('asce_version')
+        valid_asce_versions = \
+            oqvalidation.OqParam.asce_version.validator.choices
+        if asce_version not in valid_asce_versions:
+            raise ValueError(f"Valid ASCE versions are {valid_asce_versions}")
```
Comment (micheles) [inline openquake/server/views.py:874]: See before, use the validator as it is meant to be used

File: openquake/engine/aelo.py [hunk 4]
```diff
@@ -113,7 +115,11 @@ def main(lon: valid.longitude,
     This script is meant to be called from the WebUI in production mode,
     and from the command-line in testing mode.
     """
-    inputs = dict(sites='%s %s' % (lon, lat), vs30=vs30, siteid=siteid)
+    valid_asce_versions = oqvalidation.OqParam.asce_version.validator.choices
+    if asce_version not in valid_asce_versions:
+        raise ValueError(f'asce_version must be in {valid_asce_versions}')
```
[the remark this answers is not in the document]
Comment (ptormene) [reply openquake/engine/aelo.py]: Done

File: openquake/server/views.py (hunk 3 above)
Comment (ptormene) [reply openquake/server/views.py]: Done

**Comment by micheles** [comment]
Looks good, by I don't like that we are using 2 rows for the input boxes
image

I guess by reducing the descriptions of the defaults we can save horizontal space:
```
max. 5 decimal places -> max 5 decimals
fixed at 760 m/s -> 760 m/s
max. 256 characters -> max 256 chars
```

**Pull request merged**